hanami.gemspec

```diff
@@ -13,9 +14,11 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'http://hanamirb.org'
   spec.license       = 'MIT'
 
-  spec.files         = `git ls-files -c -o --exclude-standard -z -- lib/* bin/* LICENSE.md README.md CODE_OF_CONDUCT.md CHANGELOG.md FEATURES.md hanami.gemspec`.split("\x0")
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test)/})
+  spec.files         = Rake::FileList['{lib,bin}/**/{*,.*}'].exclude(*File.read('.gitignore').split)
+                                                            .reject { |f| File.directory?(f) } +
+                       %w(LICENSE.md README.md CODE_OF_CONDUCT.md CHANGELOG.md FEATURES.md hanami.gemspec)
```

Comment:
To show that the changes are basically the same:

```
irb(main):001:0> require "rake/file_list"
=> true
irb(main):002:0> a = `git ls-files -c -o --exclude-standard -z -- lib/* bin/* LICENSE.md README
.md CODE_OF_CONDUCT.md CHANGELOG.md FEATURES.md hanami.gemspec`.split("\x0") 
irb(main):003:0> b = Rake::FileList['{lib,bin}/**/{*,.*}'].exclude(*File.read('.gitignore').spl
it).reject { |f| File.directory?(f) } + %w(LICENSE.md README.md CODE_OF_CONDUCT.md CHANGELOG.md
 FEATURES.md hanami.gemspec)
irb(main):004:0> a.sort == b.sort
=> true

```

which basically implies that this solution is indeed the same, just drops the unnecessary need to use `git` in the gemspec file.